README.md

```diff
@@ -38,7 +38,19 @@ The app requires a back-end to generate [Twilio access tokens](https://www.twili
 The passcode will expire after one week. To generate a new passcode:
 
 1. Run `twilio rtc:apps:video:delete`.
-1. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+2. Run `twilio rtc:apps:video:deploy --authentication passcode`.
+
+#### Troubleshooting The Twilio CLI
+
+If any errors occur after running the `twilio rtc:apps:video:deploy --authentication passcode` command, then run the following commands:
+
+```
+twilio plugins:update
+twilio rtc:apps:video:delete
+twilio rtc:apps:video:deploy --authentication passcode
+```
+
+The commands in the above snippet updates the rtc plugin to the latest version, deletes any problematic authentication servers, and deploys a new authentication server.
```

Comment:
There seem to be 2 different styles used. Here we list out all the commands and then describe what they all do. In the deploy section we specify the command and describe what it does on the same line as needed. Not a big deal but probably better to be consistent unless there is a reason not to. It seems like the "Run abc command to do blah" approach works in more situations. Again pretty minor but thinking probably best to follow other pattern.